src/KinesisVideoStream.cpp

```diff
@@ -185,6 +185,16 @@ bool KinesisVideoStream::putFragmentMetadata(const std::string &name, const std:
     return true;
 }
 
+bool KinesisVideoStream::putEventMetadata(uint32_t event, PStreamEventMetadata pStreamEventMetadata){
+    STATUS status = ::putKinesisVideoEventMetadata(stream_handle_, event, pStreamEventMetadata);
+    if (STATUS_FAILED(status)) {
```

Comment:
any example scenario(s) when this could happen?

Reply:
That's the status code returned by the C function, so it will be all possible status codes that can be returned from [here](https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/65e38dac9b30523d43a57bc009d679e627b58d9a/src/client/src/Client.c#L972) that indicate failure